pkgs/development/python-modules/aiometer/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "aiometer";
+  version = "0.4.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "florimondmanca";
+    repo = "aiometer";
+    rev = version;
+    hash = "sha256-Db6AWNWZ5AhMFrbjBRVOFlDpzegsqNL3k5dmTHrjSi8=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.setuptools-scm
+    python3.pkgs.wheel
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    anyio
+    typing-extensions
+  ];
+
+  pythonImportsCheck = [ "aiometer" ];
+
+  meta = with lib; {
+    description = "A Python concurrency scheduling library, compatible with asyncio and trio";
+    homepage = "https://github.com/florimondmanca/aiometer";
+    changelog = "https://github.com/florimondmanca/aiometer/blob/${src.rev}/CHANGELOG.md";
+    license = licenses.mit;
+    maintainers = with maintainers; [ gwg313 ];
```

Comment:
Seems like `gwg313` is not defined in `maintainers/maintainer-list.nix`. Please add yourself in a separate commit titled `maintainers: add gwg313`, preceeding the aiometer commit.
See https://github.com/NixOS/nixpkgs/tree/master/maintainers#individual-maintainer-list